language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/TypeParameterNodeContext.java

```diff
@@ -15,44 +15,21 @@
  */
```

Comment:
Could you provide an instance where we would need to extend the TypeParameterContextProvider besides TypeParameterNodeContext

Reply:
You can refer `StreamTypeParamsNodeContext`